libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -95,7 +94,7 @@ const AP_Param::GroupInfo AC_AutoTune_Multi::var_info[] = {
     // @Description: Autotune aggressiveness. Defines the bounce back used to detect size of the D term.
     // @Range: 0.05 0.10
     // @User: Standard
-    AP_GROUPINFO("AGGR", 2, AC_AutoTune_Multi, aggressiveness, 0.1f),
+    AP_GROUPINFO("AGGR", 2, AC_AutoTune_Multi, aggressiveness, 0.05f),
```

Comment:
like it!

Reply:
Yes, I believe the improved filtering lets us do this now.

Reply:
It is also a little more conservative in that it is less likely to give a bad successful result.